slopmop/checks/security/__init__.py

```
@@ -586,6 +629,28 @@ def superseded_by(self) -> Optional[str]:
+            + (
+                Requirement(
+                    kind="python",
+                    name="pip-audit",
+                    # Invoked as ``python -m pip_audit``, so detection must probe
+                    # the module, not a ``pip-audit`` script that may not exist.
+                    probe="import",
+                    import_name="pip_audit",
+                    optional=True,
+                    reason="audits installed dependencies for known CVEs (OSV database)",
+                ),
+            )
```

Ruff (0.15.17)

[warning] Consider iterable unpacking instead of concatenation

(RUF005)